implementations/rust/ockam/ockam_command/src/node/create.rs

```diff
@@ -42,7 +45,7 @@ after_long_help = docs::after_help(AFTER_LONG_HELP)
 pub struct CreateCommand {
     /// Name of the node or a configuration to set up the node.
     /// The configuration can be either a path to a local file or a URL.
-    #[arg(value_name = "NAME_OR_CONFIGURATION", hide_default_value = true, default_value_t = random_name())]
+    #[arg(value_name = "NAME_OR_CONFIGURATION", hide_default_value = true, default_value = DEFAULT_NODE_NAME)]
```

Comment:
Maybe I would not bother with a fake name and just use `Option<String>` here.

Reply:
I need to investigate the side effects of doing that. I'm not sure it won't break things if we define this argument as optional. I'll try to tackle this in a separate PR in isolation

Reply:
About this, I tried and it adds a different complexity... Making the argument optional generates a new logical path around how you want to read or replace the arg value, and it's pretty error-prone. I don't think it's worth the effort right now.